### PR TITLE
Add makefile support for compiling scss

### PR DESCRIPTION
**Pull request by zwade**
adds `scss` target and `all` target for building before pushing to appengine

### CODE REVIEW

**Comment by zwade** [comment]
[Resolves #37]

File: Makefile [hunk 1]
```diff
@@ -5,5 +5,17 @@ MIN_JS:=static/js/app.min.js
...
+SCSS_SRC:=$(shell find static/scss -name '*.scss')
+PYSCSS:=$(shell which pyscss >/dev/null && echo `which pyscss` || echo cat)
+
 $(MIN_JS): $(JS_SRC)
 	$(MINIFY) $^ > $@
+
+scss:
+	for i in $(shell ls static/scss/); do \
+		echo "Making $$i";\
+		$(PYSCSS) -o static/css/$${i%.scss}.css static/scss/$$i;\
```
Comment (k9delight) [inline Makefile]: cat -o = invalid option -- o

since it's a deploy script, it's fine to error out if it's not installed IMO

Comment (Matir) [inline Makefile]: Agreed, the output of scss != output of cat.  (Minifying js is optional, it's okay to fail.)

File: Makefile [hunk 2]
```diff
@@ -5,5 +5,17 @@ MIN_JS:=static/js/app.min.js
...
+	done
+
+all: $(MIN_JS) scss
```
Comment (Matir) [inline Makefile]: Please make 'all' the first target.

File: Makefile (hunk 1 above)
Comment (zwade) [inline Makefile]: Resolved by 5a5e149

**Pull request merged**